packages/react-renderer-demo/src/editor-demo/docs-components/component-api/component-api.md

```diff
@@ -0,0 +1,198 @@
+&nbsp;&nbsp;timepicker/datepicker <br/>
+<br />
+
+Others: <br/>
+&nbsp;&nbsp;subform <br/>
+&nbsp;&nbsp;tab/tab item <br/>
+&nbsp;&nbsp;wizard <br/>
+
+### FormGroup wrapped components
+
+Basic components are wrapped in formGroup. FormGroup provides a standard API to show labels, helper texts, errors, etc.
+
+All those components provides a shared group of props:
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| label     | node | A label of the field |
+| description | node | Description of the field |
+| helperText      | node      |   Helper text: format, hint, etc. |
+| hideLabel | boolean | To hide label |
+| isRequired | boolean     | Is the field required? |
+| isDisabled | boolean     | Is the field disabled? |
+| isReadOnly | boolean     | Is the field readOnly? |
+
+#### Text field
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| placeholder     | node | A placeholder |
+
+<div style={{ float: 'right'}}>
+    <Button><Link component={NavLink} to={ `/component-example/text-field` }>To example</Link></Button>
+</div>
+<br/>
+
+#### Text area
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| placeholder     | node | A placeholder |
+
+<div style={{ float: 'right'}}>
+    <Button><Link component={NavLink} to={ `/component-example/text-area` }>To example</Link></Button>
+</div>
+<br/>
+
+#### Select
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| placeholder     | node | A placeholder |
+| options     | array | Options in format of { label: 'Label', value: value } |
+| loadOptions     | function | A function for loading a options asynchronously |
+| loadingMessage     | node | A message which will shown as a placeholder during the loading |
+| simpleValue | boolean | Simple value (no multi) |
+| isMulti| boolean | Allows to choose more options |
+| isClearable | boolean | Allows to clear the selected option |
+| isSearchable | boolean | Allows to search in the options |
+
+<div style={{ float: 'right'}}>
+    <Button><Link component={NavLink} to={ `/component-example/select` }>To example</Link></Button>
+</div>
+<br/>
+
+#### Checkbox
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| options     | array | Options in format { label: 'Label', value: value }, it will make it multiple a choice list. (optional)|
+
+<div style={{ float: 'right'}}>
+    <Button><Link component={NavLink} to={ `/component-example/checkbox` }>To example</Link></Button>
+</div>
+<br/>
+
+#### Radio
+
+| Prop        | Type           | Description  |
+| ------------- |:-------------:| -----:|
+| options     | array | Options in format { label: 'Label', value: value } |
+
+<div style={{ float: 'right'}}>
+    <Button><Link component={NavLink} to={ `/component-example/radio` }>To example</Link></Button>
+</div>
+<br/>
+
+#### Datepicker
```

Comment:
Just making a note here, we don't have any API specified for PF4 just yet and MUI is probably not the same as for PF3.

Reply:
Yeah, I know. I just wanted to suggest that we could define some standard API for it and 
eventually re-implement it in other packages. What do you think? Or should I leave it for now just with the `placeholder` prop?

Reply:
We will need it so we should think about the API. But there is no clue what kind of component will PF4 come up with.